docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -850,14 +862,41 @@ Default: `false`
 
 ##### --file_expiration_value_ttl_overrides_table_ttl
 
-When set to true, allows files to expire purely based on their value-level TTL expiration time (even if it is lower than the table TTL). This is useful for times where a file needs to expire earlier than its table-level TTL would allow. If no value-level TTL metadata is available, then table-level TTL will still be used. 
+When set to true, files expire based on their value-level TTL expiration time, even if it is lower than the table TTL. This is useful when a file needs to expire earlier than its table-level TTL would allow. If no value-level TTL metadata is available, then table-level TTL is still used.
 
 {{< warning title="Warning">}}
 Use of this flag can potentially result in expiration of live data - use at your discretion.
 {{< /warning >}}
 
 Default: `false`
 
+## Using postgresql.conf
```

Comment:
I'm not too sure if the usage of this file is something we want to actively expose to customers. Afaik, users can apply this flag (https://docs.yugabyte.com/latest/reference/configuration/yb-tserver/#ysql-pg-conf) to set postgres settings. I think there is a gflag as well that customers can use. I think we should confirm with someone else whether it makes sense to have line 875. 
For line 873, maybe change it to "Configuring Postgres"
For line 877, maybe change it to "For information on available Postgres settings, refer to [Server Configuration](https://www.postgresql.org/docs/11/runtime-config.html) in the PostgreSQL documentation."

Reply:
cc @sushantrmishra

Reply:
I would suggest to remove mentions postgres.conf since we use the tserver flags etc to modify the the config rather then directly modifying the content of it.

I would  suggest to put the headline as Postgres Logging options . 

Rest of the content looks good.